tools/PI/DevHome.PI/Program.cs

```diff
@@ -47,6 +47,19 @@ public static void Main(string[] args)
             .ReadFrom.Configuration(configuration)
             .CreateLogger();
 
+        var stopEvent = new EventWaitHandle(false, EventResetMode.ManualReset, $"DevHomePI - {Environment.ProcessId}");
+        var stopEventThread = new Thread(() =>

```

Comment:
prefer using threadpool see https://learn.microsoft.com/en-us/dotnet/api/system.threading.threadpool.registerwaitforsingleobject?view=net-8.0

Reply:
updated to use threadpool, but not using RegisteredWaitHandle.